app/helpers/forbidden_words_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ForbiddenWordsHelper
```

Comment:
What this helper doing? What's the reason behind it? :)

Reply:
He was added automatically (as well as .coffee files, etc). I can destroy it if you want.

Reply:
I didn't even notice - let's remove it if there is a low probablity that it is going to be needed :)